### DISCUSSION TITLE
Inserting holdings has no effect

### DISCUSSION THREAD

**Discussion by tkloy24** [flybywiresim/aircraft#2379] [Help] [answered]
Hi all,

i'm just wondering, as entering holdings doesn't seem to work. I can enter them in the MCDU (for example at the last waypoint to fly one directly). Then it also appears in the flight plan, but the Airbus continues to fly normally.

Are holdings not implemented yet? I use the current developer version.

**Comment by tkloy24** [comment]
I now created a screenshot:

![holding](https://user-images.githubusercontent.com/71210947/102009804-3a461100-3d3a-11eb-8e50-540addcfaea5.PNG)

**Reply by peerluk** [reply]
Please implement that feature! That mod is amazing! We need the basics! :)

Thank you so much for your great work

**Comment by ghost** [comment, marked as answer]
Holds are not implemented yet.
